src/main/java/org/jboss/tools/intellij/analytics/PreloadLanguageServer.java

```diff
@@ -14,7 +14,7 @@
   private final ICookie cookies = ServiceManager.getService(Settings.class);
 
   private void attachLanguageClient(final File cliFile) {
-    final String[] EXTENSIONS = {"xml", "json", "txt"};
+    final String[] EXTENSIONS = {"xml", "json", "txt", "mod"};
```

Comment:
This information is also used in the SaAction class. It think it would be good to have a single array where the supported files are defined and that would be shared by this class and SaAction